.github/workflows/qa.yml

```diff
@@ -0,0 +1,43 @@
+name: QA
```

Comment:
It was like this originally but I recommended this to be separate because linters shouldn't block tests when there's a lot of free parallel resources. Plus when it's ~100 jobs it's way easier when at least part of them are filtered out into a separate section.

Reply:
I understand your point, I am just not sure I share your opinion here. In any case, this is a matter of opinion or taste at the end. Therefore, I am OK either way.

Reply:
@codingjoe 

> I believe this should not be a separate workflow but rather a job within a single workflow.

Personally I'm ambivalent about whether it would be in a single workflow or in a separate as long as there are not so many jobs.


> So I'd add `needs: qa` to the tests' workflow.

I'd prefer not to add `needs` as it slows down the parallel jobs since all 22 test jobs would wait at least 35 seconds until QA jobs are done.